client/js/lib/prosemirror/plugins.js

```diff
@@ -336,39 +336,57 @@ const rangeTracker = (rangeTrackerKey, schema, rangeChangeCallback = () => {}) =
  */
 const rangeCreator = (pluginKey, rangeEditingKey) => {
   let tippy = null
+
+  const handleGlobalMouseup = (view, e) => {
+    const { state } = view
+    const { selection } = state
+    const { empty, from, to, $anchor, $head } = selection
+
+    if (empty) {
+      tippy?.destroy()
+      tippy = null
+      return
+    }
+
+    const lastClick = view.lastClick || view.input.lastClick
+    const hasClickLocationChanged = (e.clientX !== lastClick.x || e.clientY !== lastClick.y)
+
+    if (hasClickLocationChanged) {
+      const existingRanges = rangeEditingKey.getState(state)
+      let positionsCovered = []
+      Object.values(existingRanges).forEach(({ from, to }) => positionsCovered.push(...range(from, to)))
+      const selectedPositions = new Set(range(from, to))
+      positionsCovered = new Set(positionsCovered)
+      const isFullyCovered = isSuperset(positionsCovered, selectedPositions)
+
+      if (isFullyCovered) {
+        tippy?.destroy()
+        tippy = null
+        return
+      }
+      tippy?.destroy()
+      tippy = createCreatorMenu(view, $anchor.pos, $head.pos)
```

Comment:
You can save one line ;-) 
```suggestion
      tippy?.destroy()

      if (isFullyCovered) {
        tippy = null
        return
      }

      tippy = createCreatorMenu(view, $anchor.pos, $head.pos)
```